lib/contractDeployer.js

```diff
@@ -0,0 +1,40 @@
+
+const fs = require('fs');
+const path = require('path');
+const { btcToWeis } = require('@rsksmart/btc-eth-unit-converter');
+const solUtils = require('./sol-utils');
+const TEST_RELEASE_BTC_CONTRACT = '../contracts/CallReleaseBtcContract.sol';
+const TEST_RELEASE_BTC_CONTRACT_NAME = 'CallReleaseBtcContract';
+const SOLIDITY_COMPILER_VERSION = 'v0.8.26+commit.8a97fa7a';
+const { sendFromCow } = require('./rsk-utils');
+
+const deployCallReleaseBtcContract = async (rskTxHelper) => {
+
+    const address = await rskTxHelper.getClient().eth.personal.newAccount('');
+    await sendFromCow(rskTxHelper, address, Number(btcToWeis(0.5)));
```

Comment:
I think we have a constant for 0.5 now? maybe I am wrong :)

Reply:
Good catch! Updated.